build_daemon/README.md

```diff
@@ -0,0 +1,3 @@
+# Build Daemon
```

Comment:
Not needed! Headers on readmes are redundant in all places where they are displayed.